massdriver-application/main.tf

```diff
@@ -19,12 +19,18 @@ locals {
   policies = { for p in local.app_policies : p => jsondecode(data.jq_query.policies[p].result) }
 
   base_envs = { for k, v in local.app_envs : k => jsondecode(data.jq_query.envs[k].result) }
+  # Needed by Azure k8s until Workload Identity is out of preview.
+  azure_envs = (local.is_azure && local.is_kubernetes) ? mdxc_application_identity.main[0].azure_application_identity : {}
```

Comment:
This is what I get with the code as-written.

<img width="1787" alt="Screenshot 2023-01-08 at 1 33 53 PM" src="https://user-images.githubusercontent.com/651833/211220711-235f3899-0da5-4f79-aa61-ff395d16c656.png">